.travis.yml

```diff
@@ -1,6 +1,6 @@
-dist: xenial
```

Comment:
nit: maybe we want to change to gh actions? I am not sure if travis still works?

Reply:
Agreed, I want to switch to gh actions and also publish the JAR to maven central.  I'll plan on doing this in another PR.